FAQ.md

```diff
@@ -0,0 +1,52 @@
+# FAQ - Frequently Asked Questions
+
+<!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
+
+**Table of Contents**
+
+- [FAQ - Frequently Asked Questions](#faq---frequently-asked-questions)
+  - [Installation](#installation)
+  - [Trading](#trading)
+  - [Wallet](#wallet)
+  - [Contact us](#contact-us)
+
+<!-- markdown-toc end -->
+
+## Installation
+
+Q: Do I need to use Umbrel to use Itchysats?
+A: Not at all! While [Umbrel](https://getumbrel.com) is a convenient way to use ItchySats (one-click install from the Umbrel App Store), it is possible to try out itchysats with one of the following methods:
+
+- download the itchysats taker binary straight from [github releases page](https://github.com/itchysats/itchysats/releases).
```

Comment:
```suggestion
- download the latest itchysats taker binary straight from [github](https://github.com/itchysats/itchysats/releases/latest).
```

Reply:
I don't feel strong about that one, can also show them the complete `release` page. GitHub automatically resolves `latest` to the latest release...

Reply:
I like that!